src/stores/RedisStore.ts

```diff
@@ -0,0 +1,33 @@
+import { connect } from "../../deps.ts";
+import type { Ratelimit } from "../types/types.d.ts";
+import { Store } from "./AbstractStore.ts";
+
+export class RedisStore extends Store {
+    private readonly store;
+
+    constructor(hostname?: string, port?: number) {
```

Comment:
Instead of providing `hostname` and `port` directly, I would suggest taking an `options` input and defining an interface for it such as `RedisStoreOptions` and having `hostname` and `port` as its properties.